src/ui/public/agg_types/metrics/top_hit.js

```diff
@@ -45,7 +44,8 @@ export function AggTypesMetricsTopHitProvider(Private) {
               }
             };
           } else {
-            if (field.doc_values) {
+            // https://git.io/vSPvJ
```

Comment:
Habit, linking to specific lines of specific files in specific commits is over 120 characters in many cases, no reason this can't be there though.